External/SonicUtils/include/SonicUtils/ELFLoader.h

```diff
@@ -59,11 +66,20 @@ class ELFContainer {
 
   void GetInitLocations(uint64_t GuestELFBase, std::vector<uint64_t> *Locations);
 
-  bool HasTLS() const { return TLSHeader != nullptr; }
+  bool HasTLS() const { return TLSHeader._64 != nullptr; }
```

Comment:
Shouldn't this check 32/64 bit based on elf type?

Reply:
Since this is just a pointer check, doesn't matter here.